lighthouse-core/audits/content-width.js

```diff
@@ -6,6 +6,27 @@
 'use strict';
 
 const Audit = require('./audit.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on the content size of a site compared to its viewport. This descriptive title is shown to users when the site's content is sized appropriately. */
+  title: 'Content is sized correctly for the viewport',
+  /** Title of a Lighthouse audit that provides detail on the content size of a site compared to its viewport. This descriptive title is shown to users when the site's content is not sized appropriately. */
+  failureTitle: 'Content is not sized correctly for the viewport',
+  /** Description of a Lighthouse audit that tells the user why they should care that a site's content size should match its viewport size. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'If the width of your app\'s content doesn\'t match the width ' +
```

Comment:
this is meta but i wish we just used `" "` or backticks for these strings (so no need to escape quotes) and authored it all on one line

Reply:
+1 to the backticks to avoid escaping.  -1 to all on one line.

Reply:
thanks for rating my ideas. here's some feedback on your ratings.

> +1 to the backticks to avoid escaping.

:D

> -1 to all on one line.
 
authoring and editing these strings usually results in a waterfall of copying and pasting as you fix the line length constraints we impose here.

i think we should acknowledge that line length limits are for improving code readability, at the expense of writing. however, this isn't code, these are strings. readability isn't improved with line limits here in a way that just toggling word wrap (as you are editing or reading these strings) wouldn't as well. so no gain. but authoring is a pain.

so i think we shouldn't impose the constraints of one our tools (linting line limits) when we could just use an easy feature of another one of our tools (word wrap in IDEs).